core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -522,33 +522,8 @@
    * LEVEL_METADATA_FILE
    */
   public static final String LEVEL_METADATA_FILE = "levelmetadata_";
-  /**
-   * LOAD_STATUS SUCCESS
-   */
-  public static final String STORE_LOADSTATUS_SUCCESS = "Success";
-  /**
-   * LOAD_STATUS UPDATE
-   */
-  public static final String STORE_LOADSTATUS_UPDATE = "Update";
-  /**
-   * LOAD_STATUS FAILURE
-   */
-  public static final String STORE_LOADSTATUS_FAILURE = "Failure";
-  /**
-   * LOAD_STATUS PARTIAL_SUCCESS
-   */
-  public static final String STORE_LOADSTATUS_PARTIAL_SUCCESS = "Partial Success";
-
-  /**
-   * STORE_LOADSTATUS_STREAMING
-   */
-  public static final String STORE_LOADSTATUS_STREAMING = "Streaming";
 
   /**
-   * STORE_LOADSTATUS_STREAMING
-   */
-  public static final String STORE_LOADSTATUS_STREAMING_FINISH = "Streaming Finish";
-  /**
    * LOAD_STATUS
    */
   public static final String CARBON_METADATA_EXTENSION = ".metadata";
```

Comment:
The comment is not proper.

Reply:
The next comment also need be corrected.

Reply:
fixed